app/assets/javascripts/custom.js

```diff
@@ -1,18 +1,35 @@
 $(document).ready(function(){
-  $('.slider').slider({
+
+  $(".sale-editable-width").change(function(){
+    var food_id = $(this).data("message");
+    var percentage = $(this).val();
+    var food_price = $(".price_for_"+food_id).html();
+    var food_sale_price = (1 - (percentage/100)) * food_price;
+    $(".sale_price_"+food_id).text(food_sale_price);
```

Comment:
Identifier 'food_id' is not in camel case.<br>Identifier 'food_sale_price' is not in camel case.